Task_TheFlexGame/index.html

```diff
@@ -1,15 +1,39 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Flex Game</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-
-    <!-- Start coding from the lines below for FLEX GAME TASK; DO NOT TOUCH the code above -->
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Flex Game</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+
+  <body style="margin: 0; padding: 0">
+    <div top="0">
+      <a href="../index.html">Go Back</a>
+    </div>

```

Comment:
A class 'card-container' is missing around the 'card' elements to group them together.